models/intermediate/int_sage_intacct__account_classifications.sql

```diff
@@ -1,25 +1,34 @@
+-- Please leverage the style applied here for the other models
```

Comment:
The sql style in the models is pretty inconsistent. Please follow the style guide in this model and apply to the other models in this package.